src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, Integer age) {
+        this.login = login;
+        this.password = password;
+        this.age = age;
```

Comment:
Consider validating the parameters in the constructor to ensure that a User object is always in a valid state when it's created. For example, check if 'login', 'password', and 'age' are not null and that 'age' is a positive number.